sha512sums.txt

```diff
@@ -0,0 +1,2 @@
+113729704693e94b30e2a40ddef1ebda30a622b5bba4e89472b685ab8b5d7f3ef2226c031d3b7d8c95a77684826408a93c7522543ab26bb8068dcd4004844579  assets/categories_places365.txt
+db19eaca5d1b9577e5da0981fd84c8421bb7e90181c7a727b2a795fc0e721c7f1f2ad2f3e7ec10c1fffc23cb19734b1af512bd892eebffe25553b6fbbc02975b  assets/whole_resnet18_places365_python36.pth
```

Comment:
NOTE: This file seems to indicate Python 3.6. Since we have Python 3.7 in MAX base, this file might need an update after we upgrade the base.